cpp/lib/src/SysLog.cc

```diff
@@ -0,0 +1,38 @@
+/** \file    SysLog.h
```

Comment:
This is SysLog.cc, not SysLog.h

What's the meaning of the \file tag except redundant information? Guess i will never understand.